pkg/network/controllers/vm.go

```diff
@@ -165,17 +165,17 @@ func applyDynamicIfaceRequestOnVMI(
 }
 
 func clearDetachedInterfaces(
-	specIfaces []v1.Interface,
-	specNets []v1.Network,
-	statusIfaces map[string]v1.VirtualMachineInstanceNetworkInterface,
+	ifaces []v1.Interface,
+	nets []v1.Network,
```

Comment:
I think the `spec` prefix made it clearer what these are, especially when reading the code later.
IMO there is no good reason to remove it, but I won't nitpick on this.

Reply:
Reverted the variable names to have the `spec` prefix.